static/js/controllers/contacts-content.js

```diff
@@ -166,40 +172,48 @@ var _ = require('underscore'),
           });
       };
 
-      var getInitialData = function(contactId) {
-        return $q.all([
-            DB().get(contactId),
-            Search('reports', { subjectIds: [ contactId ] })
-          ])
-          .then(function(results) {
-            var contactDoc = results[0];
-            var selected = {
-              doc: contactDoc,
-              reports: results[1]
-            };
-            selected.doc.icon = ContactSchema.get(contactDoc.type).icon;
-            selected.doc.label = ContactSchema.get(contactDoc.type).label;
-            selected.fields = selectedSchemaVisibleFields(selected);
-
-            if (contactDoc.type === 'person') {
-              return isContactPrimaryContact(contactDoc)
-                .then(function(isPrimaryContact) {
-                  selected.doc.isPrimaryContact = isPrimaryContact;
-                  return selected;
-                });
-            }
+      var getReports = function(contactDoc) {
+        var subjectIds = [ contactDoc._id ];
+        if (contactDoc.patient_id) {
+          subjectIds.push(contactDoc.patient_id);
```

Comment:
Right - if sentinel attaches the patient uuid to the report somewhere, then the reports_by_subject view can be modified to include the patient uuid not just the short id, and searching by patient_id will no longer be necessary.